MParT/MonotoneComponent.h

```diff
@@ -1003,115 +1038,115 @@ class MonotoneComponent : public ConditionalMapBase<MemorySpace>
-            unsigned int i;
-            for(i=0; i<maxIts; ++i){ // Could just be while(true), but want to avoid infinite loop
-                xlb = xub-stepSize;
-                ylb = EvaluateSingle(workspace, cache, pt, xlb, coeffs, quad, expansion);
-                if(ylb>yd){
-                    mpart::simple_swap(ylb,yub);
-                    mpart::simple_swap(xlb,xub);
-                    stepSize *= 2.0;
-                }else{
-                    break;
-                }
-            }
-            if(i>maxIts)
-                ProcAgnosticError<MemorySpace, std::runtime_error>::error("InverseSingleBracket: lower bound iterations exceed maxIts");
-
-        // We have a lower bound...
-        }else{
-            // Now find an upper bound...
-            unsigned int i;
-            for(i=0; i<maxIts; ++i){ // Could just be while(true), but want to avoid infinite loop
-                xub = xlb+stepSize;
-                yub = EvaluateSingle(workspace, cache, pt, xub, coeffs, quad, expansion);
-                if(yub<yd){
-                    mpart::simple_swap(ylb,yub);
-                    mpart::simple_swap(xlb,xub);
-                    stepSize *= 2.0;
-                }else{
-                    break;
-                }
-            }
-            if(i>maxIts)
-                ProcAgnosticError<MemorySpace,std::runtime_error>::error("InverseSingleBracket: upper bound calculation exceeds maxIts");
-        }
-
-        assert(ylb<yub);
-        assert(xlb<xub);
-
-        // Bracketed search
-        const double k1 = 0.1;
-        const double k2 = 2.0;
-        const double nhalf = ceil(log2(0.5*(xub-xlb)/xtol));
-        const double n0 = 1.0;
-
-        double sigma, delta, rho;
-        unsigned int it;
-        for(it=0; it<maxIts; ++it){
-
-            xb = 0.5*(xub+xlb); // bisection point
-            xf = xlb - (yd-ylb)*(xub-xlb) / (yub-ylb); // regula-falsi point
-
-            sigma = ((xb-xf)>0)?1.0:-1.0; // sign(xb-xf)
-            delta = fmin(k1*pow((xub-xlb), k2), fabs(xb-xf));
-
-            xf += delta*sigma;
-
-            rho = fmin(xtol*pow(2.0, nhalf + n0 - it) - 0.5*(xub-xlb), fabs(xf - xb));
-            xc = xb - sigma*rho;
-
-            yc = EvaluateSingle(workspace, cache, pt, xc, coeffs, quad, expansion);
-
-            if(abs(yc-yd)<ftol){
-                return xc;
-            }else if(yc>yd){
-                mpart::simple_swap(yc,yub);
-                mpart::simple_swap(xc,xub);
-            }else{
-                mpart::simple_swap(yc,ylb);
-                mpart::simple_swap(xc,xlb);
-            }
-
-            // Check for convergence
-            if(((xub-xlb)<xtol)||((yub-ylb)<ftol))
-                break;
-        };
-
-        if(it>maxIts)
-            ProcAgnosticError<MemorySpace,std::runtime_error>::error("InverseSingleBracket: Bracket search iterations exceeds maxIts");
-        return 0.5*(xub+xlb);
-    }
+    // template<typename PointType, typename CoeffsType>
```

Comment:
Working from an old branch? Not sure why this is here. Admittedly, I should've deleted the docstring as well (sorry).